### DISCUSSION TITLE
Serving static XHTML / XML from codesandbox?

### DISCUSSION THREAD

**Discussion by ghost** [codesandbox/codesandbox-client#5078] [Q&A]
Sorry for any seemingly simple questions; this is my first time actually using the website.
I had heard about codesandbox sometime last year, and it seemed promising, but I had held off on actually trying it out until now.
I didn't want to set up a whole server for such a simple task, especially as it seems that codesandbox only supports Node.js, which isn't a personal favorite of mine.
All I was looking to do was setup a simple statically hosted vanilla XHTML, CSS, JS file, not seeing any templates, I had simple forked the static HTML template.
I proceeded to adjust the index.html, to make it into valid XHTML, copied it, deleted the .html, and replaced it with an index.xhtml.

The application marked the new file as "entry," which I would assume means that it is now served at the root directory of the sandbox. (can someone point me to some documentation regarding this?)

![Screenshot 2020-11-02 at 5 42 50 PM - Edited](https://user-images.githubusercontent.com/65428781/97937613-038cdc00-1d34-11eb-98d1-17c3d07ea4a7.png)

Yet, the file seems to not actually be served, after saving the changes.
I had checked the other template files for any references to "index.html," I had found that the "package.json" had it as the value of its key "main," so I had adjusted it appropriately, still without any visible change.

My sandbox: <https://codesandbox.io/s/peaceful-ride-j35um>.

Is there a proper way to do what I am seeking to do, can someone point me in the right direction?

Update: for clarification, the file **is** served, just not at the root directory.

**Comment by garethx** [comment]
If you use the file extension .html, then the file is served as expected.

**Reply by ghost** [reply]
> If you use the file extension .html, then the file is served as expected.

Yes, but that is not the file that I am intending to serve, is it possible to statically serve the XHTML file at the root, or will I be required to set up a whole server, just to serve it at the root?

What does the "entry" mean then?

**Reply by ghost** [reply]
@garethx I presume that it is not possible to accomplish my goal here?
